semver.js

```diff
@@ -1200,6 +1200,19 @@ function validRange(range, loose) {
   }
 }
 
+exports.minVersion = minVersion;
+function minVersion(range, loose) {
+  try {
+    var rangeObj = new Range(range, loose);
+  } catch (er) {
+    return null;
+  }
+  if (!rangeObj.range) {
+    return '0.0.0';
+  }
+  return rangeObj.set[0][0].semver.version;
```

Comment:
I think this will return an incorrect value in the following cases:

```
minVersion('1.2.3 || 1.0.0') // expect 1.0.0, get 1.2.3
minVersion('<=1.2.3 >1.0.0') // expect 1.0.1, get 1.2.3
minVersion('>1.2.3') // expect 1.2.4, get 1.2.3
minVersion('>1.2.3-alpha') // <-- this one is probably impossible? 1.2.3-alpha is incorrect tho
```

Reply:
I was working on a patch for this before I saw this PR. #241 deals with these edge cases and includes testing for them situations.  `minVersion('>1.2.3-alpha')` is not impossible, I believe the correct result is `1.2.3-alpha.0` which is less than `1.2.3-beta`.